### PR TITLE
87 STORY-2678 Finish Backdraft Parity

### PR DESCRIPTION
**Pull request by colin-allen**
Fixed issue with listClass being applied to the wrong element.  Had to deviate a bit from our encapsulation of filter views to get things to work.

### CODE REVIEW

**Comment by colin-allen** [comment]
@nburwell

File: src/plugins/data_table/filter_view.js [hunk 1]
```diff
@@ -107,20 +107,6 @@ var DataTableFilter = (function(options) {
...
-
-      if (this.filter.options.length > 30) {
-        listClass = "triple";
-      } else if (this.filter.options.length > 15) {
-        listClass = "double";
-      } else {
-        listClass = "single";
-      }
-
-      this.$el.addClass(listClass);
```
Comment (nburwell) [inline src/plugins/data_table/filter_view.js]: Am I missing something where this line couldn't become `this.$('.filterMenu').addClass(listClass)` and address the issue?

Comment (nburwell) [inline src/plugins/data_table/filter_view.js]: also btw while in here I would prefer this CSS class be `filter-menu` to match CSS naming convention.

**Comment by colin-allen** [comment]
Resolved using web level css.  don't need this branch anymore.

**Pull request closed without merging**